exercises/roman-numerals/introduction.md

```diff
@@ -0,0 +1,43 @@
+# Description
+
+The Romans were a clever bunch.
+They conquered most of Europe and ruled it for hundreds of years.
+They invented concrete and straight roads and even central heating.
+But one thing they never discovered though was the number zero.
```

Comment:
Misleading. Romans had zero (`nulla`) but they didn’t need to represent it with a numeral. The numerals aren’t used for arithmetic but to represent a number. So X is sufficient to represent ten without having to do I(zero symbol). Zero the number is just `nulla` and why add a whole another symbol for a special case?

Reply:
Interesting. I'm just leaving this in from the old description.md but I sort of feel that maybe this whole section could go tbh. I'm not big on the "yay conquering everyone" vibe either, and I already got rid of the bikini bit.

Reply:
Yup, I was a Classics minor in college so I also feel obligated to point out the Greeks had both concrete and central heating for centuries before the Romans. To give credit where it's due, the Romans took both to a whole new level of course.

Reply:
> I'm just leaving this in from the old description.md but I sort of feel that maybe this whole section could go tbh. 

Totally agree! I don't think it adds much.